core/domain/fs_services.py

```diff
@@ -403,3 +405,34 @@ def copy_images(
             ('image/%s' % compressed_image_filename))
         destination_fs.copy(
             source_fs.assets_path, ('image/%s' % micro_image_filename))
+
+
+def validate_and_save_image(
```

Comment:
yes we can almost in all the places where we use save_original_and_compressed_versions_of_image function.

I have filed an issue https://github.com/oppia/oppia/issues/20554 and I will add more details to it and this can be a good-first-issue.